src/components/Breadcrumb.svelte

```diff
@@ -1,31 +1,65 @@
 <script>
   export let links;
+
+  const isPlatform = (app) => {
+    const platforms = ["ios", "android", "fenix", "amazon", "echo", "fire_tv"];
+    const match = platforms.filter((item) => app.includes(item));
+    if (match.length) return true;
+    return false;
+  };
+
+  const getPlatformLogo = (app) => {
+    if (app.includes("ios")) return "/img/app-logos/apple-breadcrumb.png";
+    if (app.includes("fenix") || app.includes("android"))
+      return "/img/app-logos/android-breadcrumb.png";
+    if (
+      app.includes("amazon") ||
+      app.includes("echo") ||
+      app.includes("fire_tv")
+    )
+      return "/img/app-logos/amazon-breadcrumb.png";
+    return undefined;
+  };
```

Comment:
We should eventually clean this up so the metadata is defined by the application itself or an annotation. That can wait though.